bundle/jsky.util/src/main/java/jsky/util/DateUtil.java

```diff
@@ -17,12 +17,12 @@
 public class DateUtil {
 
     /**
-     * Return a string with the UTC time in the format /mm/dd/yy hh:mm:ss,
+     * Return a string with the UTC time in the format yyyy-MMM-dd hh:mm:ss,
      * given the current UTC time in ms.
      */
     public static String formatUTC(long time) {
```

Comment:
Did you check how many places use this function? we maybe changing something important upstream